recipes/castorapi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "castorapi" %}
+{% set version = "0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4544ab11e1144c1a66a466f5ffc379ff1031babe6d74c0fba027f3d4fd6d3b0c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
I believe this recipe can be `noarch`, so please add `noarch: python`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

Reply:
fixed